src/playlist-loader.js

```diff
@@ -9,6 +9,7 @@ import resolveUrl from './resolve-url';
 import {mergeOptions} from 'video.js';
 import Stream from './stream';
 import m3u8 from 'm3u8-parser';
+import {window} from 'global';
```

Comment:
window is the default export from `global` or `global/window`, does this work? don't we want to do `import window from 'global/window'`? (or `import window from 'global'`)